ldms/src/sampler/netlink/netlink-notifier.c

```diff
@@ -794,7 +794,7 @@ static char *get_username(const uid_t uid, forkstat_t *ft)
 static void uid_name_info_free(forkstat_t *ft)
 {
 	size_t i;
-	if (!ft || !ft->uid_name_info)
```

Comment:
Probably accidentally included in this commit?

Reply:
You are right, it belongs in a different commit. I wanted to get it all compilable/testable...which it is. But it definitely could be cleaned up. I discovered this when I built it on AlmaLinux 9.5, which is the latest.

Reply:
I'll put this in a different commit, but the code doesn't compile with gcc 11.4 without it (AlmaLinux 9.5)